components/availability/src/Availability.svelte

```diff
@@ -89,6 +93,7 @@
   export let start_date: Date;
   export let start_hour: number;
   export let view_as: "schedule" | "list";
+  export let timezone: string;
```

Comment:
Can you make sure this prop name is alphabetically sorted in the prop lists 🙏? (here, in the `defaultValueMap`, and when we're setting `internalProps`)